sim/midas/src/main/scala/midas/passes/Dedup.scala

```diff
@@ -52,11 +47,11 @@ class DedupModules extends Transform with DependencyAPIMigration with PreservesA
     * @return A transformed Firrtl AST
     */
   def execute(state: CircuitState): CircuitState = {
-    if (state.annotations.contains(NoCircuitDedupAnnotation)) {
+    if (state.annotations.contains(NoCircuitDedupClassicAnnotation)) {
       state
     } else {
       // Don't try deduping the main module of the circuit
-      val noDedups = state.circuit.main +: state.annotations.collect { case NoDedupAnnotation(ModuleName(m, c)) => m }
+      val noDedups = state.circuit.main +: state.annotations.collect { case NoDedupAnnotation(ModuleTarget(c, m)) => m }
```

Comment:
Yay for the order of the same-typed parameters being swapped for `ModuleName` and `ModuleTarget` :trollface: